src/main/java/cloud/fogbow/as/Main.java

```diff
@@ -28,10 +30,13 @@ public void run(ApplicationArguments args) {
             // Setting up plugin
             SystemIdentityProviderPlugin systemIdentityProviderPlugin =
                     SystemIdentityProviderPluginInstantiator.getSystemIdentityProviderPlugin();
+            
+            SystemRolePlugin systemRoleProviderPlugin = new DefaultSystemRolePlugin();
```

Comment:
The indication of which plugin to use must be configured in as.conf. A PluginInstantiator similar to SystemIdentityProviderPluginInstantiator (or perhaps even the same) should be used to instantiate the appropriate plugin at runtime.

In this case, I think we should have a default plugin that does nothing (i.e. adds no roles to the token). If a plugin class is not specified in the as.conf file, then this default plugin should be used. This means that you should also rename the current DefaultSystemRolePlugin (perhaps VanillaSystemRolePlugin?)

Reply:
Done.

Reply:
Perhaps, I wasn't too clear. The idea is that the name of the class for the plugin should be in the configuration file (as it is the case for the SystemIdentityProviderPlugin), however, for the SystemRolePlugin, this property is optional. If no value is given in the conf file, then we should use the DefaultSystemRolePlugin. Otherwise, if a value is provided, then the plugin should be dinamically instantiated based on the name of the class informed in the property.

Reply:
That was added in the commits https://github.com/fogbow/authentication-service/pull/62/commits/d00ae1f694371808b44beb2d777d8454c68bd61a and https://github.com/fogbow/authentication-service/pull/62/commits/36cc40d5a3370509fde5f0f6cb5603bac548551c